src/setup.rs

```diff
@@ -40,6 +58,19 @@ async fn get_session_token() -> Result<(), AocError>
     Ok(())
 }
 
+fn get_year(matches: &ArgMatches) -> Result<i32, AocError>
```

Comment:
The setup function should still take the year as an argument, so I just moved this method inside here since it is only used here now